one-ui/ui/src/config/bench.config.ts

```diff
@@ -0,0 +1,145 @@
+import {IRolesContextInterface} from "../util/roles";
+
+class MockRolesService implements IRolesContextInterface{
+  public roles: string[] = [];
+
+  public setRoles: (roles: string[]) => void = (roles: string[]) => {
+    this.roles = roles;
+  };
+  public canReadMappings:() => boolean = () => {
+    return false;
+  };
+  public canWriteMappings:() => boolean = () => {
+    return false;
+  };
+  public canReadMatchMerge:() => boolean = () => {
+    return false;
+  };
+  public canWriteMatchMerge:() => boolean = () => {
+    return false;
+  };
+  public canReadLoadData:() => boolean = () => {
+    return true;
+  };
+  public canWriteLoadData:() => boolean = () => {
+    return true;
+  };
+  public canReadEntityModels:() => boolean = () => {
+    return false;
+  };
+  public canWriteEntityModels:() => boolean = () => {
+    return false;
+  };
+  public canReadFlows:() => boolean = () => {
+    return true;
+  };
+  public canWriteFlows:() => boolean = () => {
+    return true;
+  };
+  public canReadStepDefinitions:() => boolean = () => {
+    return false;
+  };
+  public canWriteStepDefinitions:() => boolean = () => {
+    return false;
+  };
+  public hasOperatorRole:() => boolean = () => {
+    return true;
+  };
+}
+
+const response = {"data":{"jobId": "350da405-c1e9-4fa7-8269-d9aefe3b4b9a"}, "status": 200};
+const jobResp = {
+  "data": {
+    "jobId": "350da405-c1e9-4fa7-8269-d9aefe3b4b9a",
+    "flow": "testFlow",
+    "user": "admin",
+    "lastAttemptedStep": "1",
+    "lastCompletedStep": "1",
+    "timeStarted": "2020-04-04T01:17:44.918282-07:00",
+    "timeEnded": "2020-04-04T01:17:45.012137-07:00",
+    "stepResponses": {
+      "1": {
+        "flowName": "testFlow",
+        "stepName": "failedIngest",
+        "stepDefinitionName": "default-ingestion",
+        "stepDefinitionType": "ingestion",
```

Comment:
Should the stepName and definitions be mapping since its a test for mapping story. We have to create a similar one for ingestion as well and have a separate story for that.

Reply:
It can be either, this is just mock data. The UI code for rendering mapping is not in place yet, so used ingestion

Reply:
Fair enough.